dotnet/MultiUserPub.cs

```diff
@@ -0,0 +1,6 @@
+IRealtimeChannel channel = ably.Channels.Get(
```

Comment:
can we make it `var channel = `

Reply:
I think that's less clear for an example.

Reply:
Resolving this conversation so that we can land this work as it's needed for other teams to work on tomorrow.